internal/deb/extract_test.go

```diff
@@ -157,7 +157,8 @@ var extractTests = []extractTest{{
 		"/dir/":         "dir 0755",
 		"/dir/several/": "dir 0755",
 	},
-	notCreated: []string{"/dir/"},
+	// TODO discuss in PR about this. I think this is something we want.
```

Comment:
[Comment for reviewer]: The change here is that when we have a glob like `/dir/**` we used to take the permissions from the tarball only for the subfolders matched by `**` whereas now we also use the tarball permissions for `/dir/` itself.

Reply:
Done in https://github.com/canonical/chisel/pull/134/commits/681e7029120a4d3345035c2e056c062b131cc7b2.